src/ReactionService.Validation/Reactions/CreateReactionRequestValidator.cs

```diff
@@ -21,24 +22,31 @@ public class CreateReactionRequestValidator : AbstractValidator<CreateReactionRe
     RuleFor(r => r.Name)
       .MaximumLength(20)
       .WithMessage("Name is too long.")
-      .Must(x => nameRegex.IsMatch(x.Trim()))
+      .Must(x => _nameRegex.IsMatch(x.Trim()))
       .WithMessage("Name contains invalid characters.")
       .MustAsync(async(x, _) => !await reactionRepository.DoesNameExist(x))
-      .WithMessage("Reaction name already exist.");
+      .WithMessage("Reaction with this name already exists.");
 
     RuleFor(r => r.Unicode)
       .MinimumLength(7)
       .WithMessage("Unicode is too short.");
 
     RuleFor(r => r.ReactionsGroupId)
       .MustAsync(async (x, _) => await reactionsGroupRepository.DoesExistAsync(x))
-      .WithMessage("Reaction group id does not exist.");
+      .WithMessage("This group does not exist.");
 
     RuleFor(r => r.Content)
       .SetValidator(imageContentValidator);
 
     RuleFor(r => r.Extension)
-      .Must(x => ImageFormats.formats.Contains(x) && !x.Equals(".bmp") && !x.Equals(".tga"))
+      .Must(x => ImmutableList.Create(
+        ImageFormats.jpg, 
```

Comment:
Delete a space after each comma
There are should be following margins:
----ImageFormats.gif,
----ImageFormats.webp)
--.Contains(x))
.WithMessage("Wrong image extension.");